src/home_robot/home_robot/mapping/semantic/instance_tracking_modules.py

```diff
@@ -34,6 +46,7 @@ class InstanceView:
     point_cloud: np.ndarray = None
     bounds: Tuple[np.ndarray, np.ndarray]
     category_id: Optional[int] = None
+    bounds: Optional[Tuple[np.ndarray]] = None
```

Comment:
Duplicate? I merged this separately to main

Reply:
removed